easybuild/tools/filetools.py

```diff
@@ -255,7 +255,7 @@ def download_file(filename, url, path):
 
     _log.debug("Trying to download %s from %s to %s", filename, url, path)
 
-    timeout = build_option('download_timeout')
+    timeout = float(build_option('download_timeout'))
```

Comment:
this doesn't work, since now you'll run into a crash without specifying a timeout (since the default value is `None`)

fixed in https://github.com/rjeschmi/easybuild-framework/pull/1